apps/web/.env.production

```diff
@@ -1,4 +1,5 @@
 NEXT_PUBLIC_GTAG = "G-3VJTV0DLFK"
 NEXT_PUBLIC_SNAPSHOT_BASE_URL = "https://hub.snapshot.org"
 NEXT_ASSETS_URL=https://assets-prod.vertotrade.com
-REACT_APP_IS_MAINNET=true
\ No newline at end of file
+REACT_APP_IS_MAINNET=true
+FEATURE_FLAG_API_URL='https://api.raindrop.club/api/feature-flags/35'
```

Comment:
Let's remove the `35` at the end, you should either be fetching all feature flags or add the id in the code itself to generalize the feature flag implementation